simulator/tests/simulate_disconnects.rs

```diff
@@ -0,0 +1,305 @@
+// use std::time::{Duration, Instant};
```

Comment:
@binier 
do we need this or why is it commented out?

Reply:
That test needs fixing, it's partially adjusted to new simulator, but still needs some work.